lib/item.rb

```diff
@@ -0,0 +1,37 @@
+class Item
+  attr_accessor :publish_date
+  attr_reader :id, :genre, :author, :label, :archived
+
+  def initialize(genre, author, label, date, archived: false)
+    @id = Random.rand(1..1000)
+    add_genre(genre)
+    add_author(author)
+    add_label(label)
+    @publish_date = Date.git parse_date(date)
```

Comment:
please fix `Date.git` in `@publish_date = Date.git parse_date(date)` because this doesnt contribute to any outcome.